### DISCUSSION TITLE
Tailwindcss

### DISCUSSION THREAD

**Discussion by angelalaw01887** [Megabit/Blazorise#5025] [Q&A]
I am exploring new tech blazor, tailwindcss, and blazorise for my own personal use.  when implementing tailwind I notice the instructions say use the <script src="https://cdn.tailwindcss.com"></script> I am curious does Blazorise have a way to pair this down.  Tailwind doesn't recommend using that for anything other than developers machines and recommends building a pair down version when putting anything out for public use or production use...

**Comment by stsrki** [comment]
We only recommend using it in development as it is the easiest and fastest way to start with Blazorise. We don't have any special configuration apart from it.

Otherwise, setting up the Tailwind provider should go as usual by following the instructions https://tailwindcss.com/docs/installation once the project is ready for production.
